resources/js/components/UserSidebar.js

```diff
@@ -7,9 +7,9 @@ export default class UserSidebar extends Component {
   render() {
     return (
       <div class="block">
-        <h1>{this.props.user.organization}</h1>
+        <h1>{this.props.user.organization.name}</h1>
         <div class="divide-15" />
-        <p>{this.props.user.email}</p>
+        <p>Welcome, {this.props.user.name}</p>
```

Comment:
I thought we were going to remove the username and stick with an email. I'm okay with keeping it; name would have to be returned from the server on a login.